src/main/java/org/isf/utils/sms/SetupGSM.java

```diff
@@ -46,8 +48,10 @@
 public class SetupGSM extends JFrame implements SerialPortEventListener {
 
 	private static final long serialVersionUID = 1L;
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	private static final String FILE_PROPERTIES = "GSM.properties";
```

Comment:
one more, CAPITAL

Reply:
is it clear that are physical configuration files with case also?